testbed/main.cc

```diff
@@ -33,19 +33,27 @@
 #include <GLFW/glfw3.h>
 
 #include <algorithm>
+#include <cassert>
 #include <cstdlib>
 #include <ctime>
+#include <exception>
 #include <fstream>
 #include <iostream>
 #include <iterator>
 #include <list>
+#include <numeric>
 #include <sstream>
 #include <string>
 #include <vector>
 
 using namespace std;
 using namespace p2t;
 
+using Points = vector<Point*>;
```

Comment:
Why? This makes the code harder to understand and `Points` isn't that much shorter.

Reply:
No problem in removing those using statements.
I find it clearer on the contrary to name data structures used in many places, but I understand your point of view as well. Also taking into account the fact that the poly2tri API does not provide those abstractions and we should keep a similar style in the testbed.